docs/using-injected-bindings/using-injected-bindings.md

```diff
@@ -0,0 +1,165 @@
+---
+sidebar_position: 1
+---
+
+# General Concept
+
+When using Service Binding operator, you'll get a set of binding metadatas injected into your application.
+That injection can be done in two different ways:
+- As files: this is the default behavior,
+- As environment variables: if
+`.spec.bindAsFiles` is set to `false` in the  resource.
+
+## Consuming the Bindings from Applications
+
+The Application Projection section of the spec describes how the bindings are
+projected into the application.  The primary mechanism of projection is through
+files mounted at a specific directory.  The bindings directory path can be
+discovered through `SERVICE_BINDING_ROOT` environment variable setup in your application.  
+The operator must have injected `SERVICE_BINDING_ROOT` environment to all the containers
+where bindings are created.
+
+Within this service binding root directory, there could be multiple bindings
+projected from different Service Bindings.  For example, suppose an application
+requires to connect to a database and cache server. In that case, one Service
+Binding can provide the database, and the other Service Binding can offer
+bindings to the cache server.
```

Comment:
binding metadata.

The spec uses "binding", "projected binding", and "projected binding data" interchangeably. In our docs, we use "metadata" instead of "data".